packages/grafana-prometheus/src/components/metrics-browser/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const LIST_ITEM_SIZE = 25;
```

Comment:
do we want this file in the same directory as types.ts `packages/grafana-prometheus/src/constants.ts` instead of being nested in the metrics-broeser dir?

Reply:
we can move this to `./src/constants.ts`.